src/schema/submission.yaml

```diff
@@ -164,7 +154,7 @@ classes:
       - IdentifiedByAliasMixin
       - AttributeMixin
     description: >-
-      An condition that is linked to comparable samples.
+      The Condition specifies which special characteristics and treatments apply to a Sample. It is used to link comparable Samples.
```

Comment:
"It is used to link comparable Samples" This is not clear to me. 

In the previous classes, an indefinite article was used. But here it is 'the'.

Reply:
I added a example, does that make it more clear?

Reply:
Why don't we just say smt like "The Condition specifies the special characteristics of a Sample (e.g., whether the sample comes from a disease or a healthy tissue)."?

Reply:
Moved the description to the Condition slot in the Sample class